### PR TITLE
fix: limit character with truncate

### LINKED ISSUE #223: Limitation of number of characters in profile view / limitante de cantidad de caracteres en vista de perfil
**Explanation**

- When entering the profile view in the vendor flow, there should be a character limit on what is displayed.
- Al ingresar a la vista de perfil en el flujo de proveedores, debe haber un límite de caracteres en lo que se muestra

image

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

Limit the size of the trade name with css. Now when a name is too long it will have 3 dots to shorten it.

fix #223 

#### Screenshots

chrome_lsmT9VvNqT

#### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### How to test it

1. Login provide user
2. See the header
3. Go to profile
4. edit comercial name whit many letters or words
5. See the header

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
@angelik0828 please check

**Comment by angelik0828** [comment]
## ❌ When creating a product with a very large name and saving, it did not update when clicking on edit, it took the names to the form but not in the upper right part where the name goes, if there is no logo

image

**Comment by Eghinner** [comment]
> When creating a product with a very large name and saving, it did not update when clicking on edit, it took the names to the form but not in the upper right part where the name goes, if there is no logo

Becuse in the upper right just appear the comercial name, created with provider

**Comment by angelik0828** [comment]
## It should not be, the company name will appear if there is no logo, otherwise the logo should be on the top right.

**Comment by angelik0828** [comment]
## Make the change in the supplier's name and it did not change in the products section or on the same page where you entered to change the profile.

image
image

**Comment by angelik0828** [comment]
@Eghinner can this be validated?

**Comment by Eghinner** [comment]
> @Eghinner can this be validated?

WIP

**Comment by heliomar-pena** [comment]
Is not the provider's name functionality from another PR?

This #206 I thought, please check @angelik0828 @Eghinner

File: components/BackHeader/BackHeader.module.scss [hunk 1]
```diff
@@ -65,3 +65,12 @@
...
   flex-direction: row;
 }
+
+.title {
+  overflow: hidden;
+  padding-right: 1.25rem;
+  text-align: right;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 250px;
```
Comment (heliomar-pena) [inline components/BackHeader/BackHeader.module.scss]: Please change this for a max-width, so short names aren't using more space than the needed

**Review by heliomar-pena — APPROVED** [review]

**Comment by Eghinner** [comment]
> components/BackHeader/BackHeader.module.scss

this problem can be solved in another PR

**Comment by angelik0828** [comment]
## Reviewing it can be fixed in another PR, this one already has the character limitation

image

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**